core/config/entity.go

```diff
@@ -82,7 +82,10 @@ type StatConfig struct {
 // SystemStatConfig represents the configuration items of system statistics.
 type SystemStatConfig struct {
 	// CollectIntervalMs represents the collecting interval of the system metrics collector.
-	CollectIntervalMs uint32 `yaml:"collectIntervalMs"`
+	CollectIntervalMs       uint32 `yaml:"collectIntervalMs"`
+	CollectLoadIntervalMs   uint32 `yaml:"collectLoadIntervalMs"`
```

Comment:
Could you please add some doc comments in the new fields?

Reply:
Okay